docs/maintainer-guide/issues.md

```diff
@@ -0,0 +1,58 @@
+1. **Be inquisitive.** Ask questions on the issue whenever something isn't clear. Don't assume you understand what's being reported if there are details missing. Whenever you are unsure, it's best to ask for more information.
+1. **Not all requests are equal.** It's unlikely we'll be able to accommodate every request, so don't be afraid to say that something doesn't fit into the scope of the project or isn't practical. It's better to give such feedback if that's the case.
+1. **Close when appropriate.** Don't be afraid to close issues that you don't think will be done, or when it's become clear from the conversation that there's no further work to do. Issues can always be reopened if they are closed incorrectly, so feel free to close issues when appropriate. Just be sure to leave a comment explaining why the issue is being closed (if not closed by a commit).
+
+## When an Issue is Opened
+
+When an issue is first opened, follow these steps:
+
+1. Is it clear what is being requested? If no, ask for more information. If yes, then:
+    * Label bug reports with the "bug" label (also use the "accepted" label if you can reproduce and verify the bug)
+    * Label requests for changes to existing features with the "enhancement" and "proposal" labels
+    * Label requests for completely new features with the "feature" and "proposal" labels
+    * Use an appropriate label for the part of the project the issue refers to
+1. If it's a bug, be sure the issue includes the following (ask for any missing information):
+    * The ESLint version being used
+    * The configuration being used and the actual source that reproduces the problem
+    * The actual ESLint output (not a summary, have them copy-paste it)
+    * What they expected to happen
+1. If it's an enhancement or feature request, be sure the issue mentions the primary use case (the problem they are trying to solve) and not just a suggestion for a technical change (which needs to be a solution to a problem). If the use case isn't clear, ask for more information. Frequently, "What problem are you trying to solve?" can clarify that.
+1. If it's just a question (one that would normally go to the mailing list or chatroom), feel free to answer the question and close the issue when the conversation is over.
+1. If you can verify that the issue is a duplicate of another issue, mention this in a comment and close the issue. Be sure to reference the issue number of the issue they should refer to, such as, "Duplicate of #1234."
+1. Regardless of the above, always leave a comment. Don't just add labels, engage with the person who opened the issue by asking a question (request more information if necessary) or stating your opinion of the issue. If it's a verified bug, ask if the user would like to submit a pull request.
```

Comment:
I see what you mean. My concern is that the person submitting the issue doesn't get any notification when a label is added, but does when a comment is left (so they know someone has looked at it). Maybe we need another label for "needs review"? So you can categorize but still mark as needing a response?